### PR TITLE
Allow url.path sanitization

### PR DESCRIPTION
**Pull request by klauco**
Fixes #
Currently, url.full can be sanitized, however the path part of the URL can't. This PR allows sanitization of url.path.

## Changes
-url.path is explicitely allowed to be sanitized.

## Merge requirement checklist

* [ ] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [ ] [CHANGELOG.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CHANGELOG.md) updated for non-trivial changes.
* [ ] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

**Comment by trisch-me** [comment]
hey @klauco 
first you need to sign CLA, as for the PR itself, md files are autogenerated, if you want to change content you need to change appropriate yaml file and then run `make table-generation`

I also do have questions about change you've proposed, but let's discuss it later

File: docs/attributes-registry/url.md [hunk 1]
```diff
@@ -10,13 +10,15 @@ linkTitle: URL
...
 | `url.scheme` | string | ![Stable](https://img.shields.io/badge/-stable-lightgreen)<br>The [URI scheme](https://www.rfc-editor.org/rfc/rfc3986#section-3.1) component identifying the used protocol. | `https`; `ftp`; `telnet` |
 
 **[1]:** For network calls, URL usually has `scheme://host[:port][path][?query][#fragment]` format, where the fragment is not transmitted over HTTP, but if it is known, it SHOULD be included nevertheless.
 `url.full` MUST NOT contain credentials passed via URL in form of `https://username:password@www.example.com/`. In such case username and password SHOULD be redacted and attribute's value SHOULD be `https://REDACTED:REDACTED@www.example.com/`.
 `url.full` SHOULD capture the absolute URL when it is available (or can be reconstructed) and SHOULD NOT be validated or modified except for sanitizing purposes.
 
-**[2]:** Sensitive content provided in query string SHOULD be scrubbed when instrumentations can identify it.
+**[2]:** `url.path` SHOULD NOT be validated or modified except for sanitizing purposes.
```
Comment (lmolkova) [inline docs/attributes-registry/url.md:21]: please add this note in the yaml definition and re-generate the markdown (it's auto-generated).
Also, perhaps it makes sense to add the same note as for query?
`Sensitive content provided in the path SHOULD be scrubbed when instrumentations can identify it.` ?

Comment (klauco) [reply docs/attributes-registry/url.md:21]: The not was derived from url.full. Perhaps all three attributes url.full, url.path and url.query should have the same note? - 

_"<attribute> SHOULD NOT be validated or modified except for sanitizing purposes. Sensitive content SHOULD be scrubbed when instrumentations can identify it."_

Comment (lmolkova) [reply docs/attributes-registry/url.md:21]: yes, having the same note would be great.

Comment (klauco) [reply docs/attributes-registry/url.md]: updated

**Review by lmolkova — APPROVED** [review]

File: .chloggen/allow-sanitizing-url-path.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: url
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Allowed sanitization of `url.path` component of the URL.
```
Comment (trask) [inline .chloggen/allow-sanitizing-url-path.yaml:13]:
```suggestion
note: Sensitive content provided in `url.full`, `url.path`, and `url.query` SHOULD be scrubbed when instrumentations can identify it.
```

**Review by trask — APPROVED** [review]

**Review by joaopgrassi — APPROVED** [review]

**Comment by joaopgrassi** [comment]
@klauco can you please fix the conflicts? Thanks!

**Pull request merged**